src/components/customGreeting.vue

```diff
@@ -15,7 +15,7 @@
     >
       <h3>About this directory</h3>
       <p>This tool can help you find resources that help Black males in Philadelphia. You can:</p>
```

Comment:
Change copy to:
This tool can help you find resources that assist Black males in Philadelphia.